azure-mgmt-website/src/main/java/com/microsoft/azure/management/website/AppServiceCertificate.java

```diff
@@ -6,28 +6,183 @@
 
 package com.microsoft.azure.management.website;
 
-import com.microsoft.azure.management.resources.fluentcore.arm.models.IndependentChildResource;
+import com.microsoft.azure.management.resources.fluentcore.arm.models.GroupableResource;
+import com.microsoft.azure.management.resources.fluentcore.model.Creatable;
+import com.microsoft.azure.management.resources.fluentcore.model.Refreshable;
 import com.microsoft.azure.management.resources.fluentcore.model.Wrapper;
-import com.microsoft.azure.management.website.implementation.AppServiceCertificateInner;
+import com.microsoft.azure.management.website.implementation.CertificateInner;
+import org.joda.time.DateTime;
+
+import java.io.File;
+import java.util.List;
 
 /**
- * An immutable client-side representation of an Azure App Service Certificate.
+ * An immutable client-side representation of an Azure app service certificate.
  */
 public interface AppServiceCertificate extends
-        IndependentChildResource,
-        Wrapper<AppServiceCertificateInner> {
+        GroupableResource,
+        Refreshable<AppServiceCertificate>,
+        Wrapper<CertificateInner> {
+    /**
+     * @return the friendly name of the certificate
+     */
+    String friendlyName();
+
+    /**
+     * @return the subject name of the certificate
+     */
+    String subjectName();
+
+    /**
+     * @return the host names the certificate applies to
+     */
+    List<String> hostNames();
+
+    /**
+     * @return the pfx blob
+     */
+    String pfxBlob();
+
     /**
-     * @return the key vault resource Id
+     * @return the app name
      */
-    String keyVaultId();
+    String siteName();
 
     /**
-     * @return the key vault secret name
+     * @return the self link
      */
-    String keyVaultSecretName();
+    String selfLink();
 
     /**
-     * @return the status of the Key Vault secret
+     * @return the certificate issuer
      */
-    KeyVaultSecretStatus provisioningState();
+    String issuer();
+
+    /**
+     * @return the certificate issue Date
+     */
+    DateTime issueDate();
+
+    /**
+     * @return the certificate expriration date
+     */
+    DateTime expirationDate();
+
+    /**
+     * @return the certificate password
+     */
+    String password();
+
+    /**
+     * @return the certificate thumbprint
+     */
+    String thumbprint();
+
+    /**
+     * @return if the certificate valid
+     */
+    Boolean valid();
+
+    /**
+     * @return the raw bytes of .cer file
+     */
+    String cerBlob();
```

Comment:
Cer is not certificate here: https://en.wikipedia.org/wiki/Certified_Emission_Reduction